docs/preview/03-Features/security/auth/shared-access-key.md

```diff
@@ -79,6 +67,8 @@ builder.Services.AddControllers(options =>
 For this setup to work, an Arcus secret store is required as the provided secret name (in this case `"shared-access-key-name"`) will be looked up.
 See [our official documentation](https://security.arcus-azure.net/features/secret-store/) for more information about setting this up.
 
+> 💡 This authentication mechanism supports multiple secret versions so that many different versions of a single configured secret are checked during authentication. See [our official documentation]() for more information on how to configured your secret as 'versioned' in the secret store.
```

Comment:
Build failed bc we can't fill this out yet, as the security docs are not yet released: https://github.com/arcus-azure/arcus.security/pull/363